library/cwm/src/modules/CWM.rb

```diff
@@ -726,7 +725,9 @@ def CreateWidgets(names, source)
         deep_copy(m)
       end
       ret = Builtins.maplist(ret) { |w| prepareWidget(w) }
-      deep_copy(ret)
+      r = deep_copy(ret)
```

Comment:
I think this deepcopy is not needed, as above is called maplist which itself call deepcopy - https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/builtins.rb#L121